test/server/kill-test.js

```diff
@@ -0,0 +1,136 @@
+const __importDefault = this && this.__importDefault || function (mod) {
+    return mod && mod.__esModule ? mod : { 'default': mod };
+};
+const sinon     = require('sinon');
+const assert    = require('assert');
+const osFamily = __importDefault(require('os-family'));
+const utilsProcess = require('../../lib/utils/process');
+const childProcess = require('child_process');
+
+describe('PROCESS KILLER', () => {
+    const BROWSER_ID = 1;
+
+    if (osFamily.default.win) {
+        describe('Windows process killer', () => {
+            it('Should call process.kill()', async () => {
+                let winProcessKilled = false;
+
+                const stubKill = sinon.stub(process, 'kill').callsFake(() => {
+                    winProcessKilled = true;
+                });
+
+                await utilsProcess.killBrowserProcess(BROWSER_ID);
+
+                stubKill.restore();
+                assert.deepStrictEqual(winProcessKilled, true);
+            });
+        });
+    }
+    else {
+        describe('Unix process killer', () => {
+            const CHECK_KILLED_DELAY = 2000;
+
+            const stubChildProcess = {
+                stdout: {
+                    on: () => {}
+                },
+                stderr: {
+                    on: () => {}
+                },
+                on: (event, listener) => {
+                    if (event === 'exit')
+                        listener();
+                }
+            };
+
+            it('Should try simple kill and stop if it works', async function () {
+                // After killing the process, program should wait some time before checking whether it is killed
+                // In test we should wait a little bit more
+                this.timeout(1000 + CHECK_KILLED_DELAY);
+
+                let getInfoCount = 0;
+
+                stubChildProcess.stdout.on = (event, listener) => {
+                    if (event === 'data' && getInfoCount === 0) {
+                        listener('1 1');
+                        getInfoCount++;
+                    }
+                };
+
+                let unixProcessKilled = false;
+
+                const stubSpawn = sinon.stub(childProcess, 'spawn').returns(stubChildProcess);
+                const stubKill = sinon.stub(process, 'kill').callsFake((processId, flag) => {
```

Comment:
We separate multiline assignments from the singleline ones:
```suggestion
                const stubSpawn = sinon.stub(childProcess, 'spawn').returns(stubChildProcess);
                
                const stubKill = sinon.stub(process, 'kill').callsFake((processId, flag) => {
```